packages/react/src/dialog-title/DialogTitle.tsx

```diff
@@ -21,13 +21,13 @@ export const DialogTitle = forwardRef<HTMLDivElement, DialogTitleProps>(
 
     return (
       <Flex gap="xs" pb="md" pt="lg" px="lg" {...sprinkleProps}>
-        <Heading asChild level="4">
+        <Heading asChild fontSize="3xl" fontWeight="500" level="4">
```

Comment:
if this is a level 3 heading then let's use that - so for `Heading` we should avoid modifying the `fontSize` directly as much as possible and rely on the `level`/`variant` prop instead
```suggestion
        <Heading asChild fontWeight="500" level="3">
```